mozilla-services/socorro
# bug 1303758: remove cpu_name

This finishes the transition from "cpu_name" to "cpu_arch". The cpu_arch
field holds the architecture the binaries were built for.

To test:

1. process some crashes
2. verify the "Build Architecture" and "CPU Count" fields have correct values in the report view
3. verify the bugzilla links are built properly
4. verify that the `cpu_arch` and `cpu_count` fields can be searched and faceted on
